### PR TITLE
Add an endpoint to view basic stats

### PR DESCRIPTION
**Pull request by sjohnson31**

### CODE REVIEW

File: api/src/room_store/memory_room_store.py [hunk 1]
```diff
@@ -165,3 +165,10 @@ async def get_room_idle_seconds(self, room_id: str) -> int:
         if room_id not in self.storage.rooms_by_id:
             raise NoSuchRoomError
         return int(time.time()) - self.storage.last_room_activity_by_id[room_id]
+
+    async def seconds_since_last_activity(self) -> int:
+        most_recent_activity = 0
+        for _, last_acitivty_time in self.storage.last_room_activity_by_id.items():
```
Comment (sburba) [inline api/src/room_store/memory_room_store.py:171]: acitivty -> activity

File: api/tests/test_rate_limit.py [hunk 2]
```diff
@@ -406,3 +406,11 @@ async def test_release_connection_on_exception(rate_limiter: RateLimiter) -> Non
 
     # Should be allowed because the connection should be freed after the exception
     await rate_limiter.acquire_connection('user-last', 'room-1')
+
```
Comment (sburba) [inline api/tests/test_rate_limit.py:409]: I think we need tests for zero connections and tests for connections spread across multiple users

File: api/tests/test_room_store.py [hunk 3]
```diff
@@ -244,3 +244,13 @@ async def test_write_if_missing_does_not_overwrite(room_store: RoomStore) -> Non
     await room_store.add_request(TEST_ROOM_ID, VALID_REQUEST)
     await room_store.write_if_missing(TEST_ROOM_ID, [ANOTHER_VALID_ACTION])
     assert list(await room_store.read(TEST_ROOM_ID)) == [VALID_ACTION]
+
```
Comment (sburba) [inline api/tests/test_room_store.py:247]: I think we need tests for zero rooms and tests for activity time for multiple rooms

File: api/src/room_store/room_store.py [hunk 4]
```diff
@@ -98,3 +98,6 @@ async def get_room_idle_seconds(self, room_id: str) -> int:
         :return: Number of seconds since the room was read or written to by a user
         """
         ...
+
+    async def seconds_since_last_activity(self) -> int:
```
Comment (sburba) [inline api/src/room_store/room_store.py:102]: docs

File: api/src/rate_limit/rate_limit.py [hunk 5]
```diff
@@ -72,3 +72,6 @@ async def acquire_new_room(self, user_id: str) -> None:
         many rooms recently
         """
         ...
+
+    async def get_total_num_connections(self) -> int:
```
Comment (sburba) [inline api/src/rate_limit/rate_limit.py:76]: docs

**Review by sburba — APPROVED** [review]
Neato
